src/qt/managenamespage.cpp

```diff
@@ -0,0 +1,324 @@
+#include "managenamespage.h"
+#include "ui_managenamespage.h"
+
+#include "base58.h"
+#include "configurenamedialog.h"
+#include "csvmodelwriter.h"
+#include "guiutil.h"
+#include "names/common.h"
+#include "nametablemodel.h"
+#include "platformstyle.h"
+#include "ui_interface.h"
+#include "util.h"
+#include "validation.h" // cs_main
+#include "wallet/wallet.h"
+#include "walletmodel.h"
+
+#include <univalue.h>
+
+#include <QMessageBox>
+#include <QMenu>
+#include <QSortFilterProxyModel>
+
+ManageNamesPage::ManageNamesPage(const PlatformStyle *platformStyle, QWidget *parent) :
+    QWidget(parent),
+    platformStyle(platformStyle),
+    ui(new Ui::ManageNamesPage),
+    model(0),
```

Comment:
Again, please use `nullptr` instead of `0` (or `NULL`, for that matter).  I won't comment this anymore.